buildSrc/src/main/kotlin/deps.kt

```diff
@@ -74,12 +69,14 @@ object Spine {
     const val server = "io.spine:spine-server:${Versions.spineCore}"
     const val client = "io.spine:spine-client:${Versions.spineCore}"
     const val core = "io.spine:spine-core:${Versions.spineCore}"
+    const val serverTestUtil = "io.spine:spine-testutil-server:${Versions.spineCore}"
```

Comment:
I think this dependency comes together with `spine.enableJava().server()`. Please check.